bundles/com.espressif.idf.core/src/com/espressif/idf/core/build/messages.properties

```diff
@@ -29,7 +29,7 @@ IDFBuildConfiguration_PreCheck_DifferentIdfPath=The project was built using the
 IDFToolChainsMissingErrorMsg=Toolchains are missing. Please use ESP-IDF Manager for configuring
 
 NoActiveEspIdfInWorkspaceMsgTitle=ESP-IDF Setup
-NoActiveEspIdfInWorkspaceMsg=ESP-IDF is required to use Espressif IDE. Would you like to configure it now?
+NoActiveEspIdfInWorkspaceMsg=ESP-IDF tools installed via EIM have been detected, but this workspace is not configured yet. The IDE needs to complete setup to enable full functionality. Would you like to proceed now?
```

Comment:
.... The IDE needa to complete setup to enable full functionality.

Now thinking this seems redundant.
How about just keep first sentence and then Would you like to setup now?